udf/add_monthly_searches.sql

```diff
@@ -0,0 +1,129 @@
+/*
+*/
+
+CREATE TEMP FUNCTION
+  udf_add_monthly_searches(prev ARRAY<STRUCT<key STRING, value STRUCT<total_searches ARRAY<INT64>, tagged_searches ARRAY<INT64>, search_with_ads ARRAY<INT64>, ad_click ARRAY<INT64>>>>,
+                           curr ARRAY<STRUCT<key STRING, value STRUCT<total_searches ARRAY<INT64>, tagged_searches ARRAY<INT64>, search_with_ads ARRAY<INT64>, ad_click ARRAY<INT64>>>>,
+                           submission_date DATE) AS ((
+  WITH prev_tbl AS (
+    SELECT *
+    FROM UNNEST(prev) AS p
```

Comment:
the ` AS p` here is unused:
```suggestion
    FROM UNNEST(prev)
```